CONTRIBUTING.md

```diff
@@ -125,41 +148,62 @@ Please follow these steps to have your contribution considered by the maintainer
+
+Documentation is buit using [Docusauras](https://docusaurus.io/) from this [GitHub repository](https://github.com/FOSSBilling/docs). You can contribute directly to the repo on GitHub or using the *Edit this page* links on each page of the docs site.
+
+Please try to be thorough and clear when writing directions. Something might seem obvious to you, but do not assume that it is to everybody else. 
+
+### Translating FOSSBilling
+
+We would like FOSSBilling to be available to as many people in as many languages as possible. 
+
+The software is primarily written in English. If you are a native or fluent speaker of another language then we could really use your help with transaltionn. 
+
+### Sponsor the project
+
+If you do not have the time or necessary skills to actively take part in the developemnt or documentation of the project then you can still play a part by making a financial contribution to the project. This could be a one time contribution or a recurring montly donation.
+
+You can do this using [GitHub Sponsors](https://github.com/sponsors/FOSSBilling) or on [Open Collective](https://opencollective.com/fossbilling). 
+
 
 ## Styleguides
 
-### Git Commit Messages
+### Commit Messages Styleguide
+
+Please be as clear and descriptive as possible in your commit messages, it makes it much easier for everyone to follow them. 
 
 * Use the present tense ("Add feature" not "Added feature")
 * Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
 * Limit the first line to 72 characters or less
 * Reference issues and pull requests liberally after the first line
 * When only changing documentation, include `[ci skip]` in the commit title
-* Consider starting the commit message with an applicable emoji:
-    * :art: `:art:` when improving the format/structure of the code
-    * :racehorse: `:racehorse:` when improving performance
-    * :non-potable_water: `:non-potable_water:` when plugging memory leaks
-    * :memo: `:memo:` when writing docs
-    * :penguin: `:penguin:` when fixing something on Linux
-    * :apple: `:apple:` when fixing something on macOS
-    * :checkered_flag: `:checkered_flag:` when fixing something on Windows
-    * :bug: `:bug:` when fixing a bug
-    * :fire: `:fire:` when removing code or files
-    * :green_heart: `:green_heart:` when fixing the CI build
-    * :white_check_mark: `:white_check_mark:` when adding tests
-    * :lock: `:lock:` when dealing with security
-    * :arrow_up: `:arrow_up:` when upgrading dependencies
-    * :arrow_down: `:arrow_down:` when downgrading dependencies
-    * :shirt: `:shirt:` when removing linter warnings
-
+* When it is appropriate start the commit message with an applicable emoji:
+    * 🎨 when improving the format/structure of the code
+    * 🚀 when improving performance
+    * 📝 when writing docs
+    * 🐛 when fixing a bug
+    * 🔒 when dealing with security
+    * ⬆️ when upgrading dependencies
+    
```

Comment:
If we are going with the icons we should have an icon for a new feature ...

Reply:
Good point - not sure what icon fits best though.

Reply:
Updated